server/index.js

```diff
@@ -0,0 +1,5 @@
+const app = require("./app");
+const port = process.env.PORT || "3000";
+app.set("port", port);
```

Comment:
Just a minor change here, for the purpose of following conventions its better to write environment variables in all capital letters. So `port` should be `PORT`. This communicates where this variable lives more clearly.